stable/artifactory-ha/ReverseProxyConfiguration.md

```diff
@@ -10,33 +10,76 @@ the required configuration snippet which you can then download and install direc
 **Note**: Nginx image distributed with Artifactory helm chart is custom image managed and maintained by JFrog.  
   
 ####  Features of Artifactory Nginx
-*   Provides default configuration with self signed SSL certificate.
-*   Auto update reverse proxy configuration by pulling configuration generated by Reverse Proxy Configuration Generator from Artifactory using [REST API](https://www.jfrog.com/confluence/display/RTF/Configuring+a+Reverse+Proxy#ConfiguringaReverseProxy-RESTAPI)
+*   Provides default configuration with self signed SSL certificate generated on each helm install/upgrade.
 *   Persist configuration and SSL certificate in `/var/opt/jfrog/nginx` directory
   
-#### Steps to use Reverse Proxy Configuration Generator to configuration for reverse proxy in nginx (Artifactory Pro/Enterprise).  
-*   Deploy Artifactory using helm chart with Nginx enabled.
-*   Go to Artifactory Admin -> HTTP Settings. 
-    Example values: 
-    ```
-    Docker Access Method: SubDomain
-    Server Provider: Nginx
-    Internal Hostname: $ARTIFACTORY_SERVICE_NAME (Get Artifactory Service Name by running `kubectl get svc` command)
-    Public Server Name: $DOMAIN_NAME
-    SSL Key Path: /var/opt/jfrog/nginx/ssl/tls.key (If SSL Cert is provided via Secret)
-    SSL Certificate Path: /var/opt/jfrog/nginx/ssl/tls.crt (If SSL Cert is provided via Secret)
-    ```
-*   Provide appropriate values and save configuration.
-*   Once configuration is saved Nginx will automatically fetch reverse proxy configuration snippet from Artifactory and apply it immediately.
-  
+#### Changing the default Artifactory nginx conf
+Use a values.yaml file for changing the value of nginx.mainConf or nginx.artifactoryConf
+These configuration will be mounted to the nginx container using a configmap.
+For example:
+1. Create a values file `nginx-values.yaml` with the following values:
+```yaml
+nginx:
+  artifactoryConf: |
+    ssl_protocols TLSv1 TLSv1.1 TLSv1.2 TLSv1.3;
+    ssl_certificate  {{ .Values.nginx.persistence.mountPath }}/ssl/tls.crt;
+    ssl_certificate_key  {{ .Values.nginx.persistence.mountPath }}/ssl/tls.key;
+    ssl_session_cache shared:SSL:1m;
+    ssl_prefer_server_ciphers   on;
+    ## server configuration
+    server {
+      listen 443 ssl;
+      listen 80;
+      ## Change to you DNS name you use to access Artifactory 
+      server_name ~(?<repo>.+)\.jfrog.team jfrog.team;
+
+      if ($http_x_forwarded_proto = '') {
+        set $http_x_forwarded_proto  $scheme;
+      }
+      ## Application specific logs
+      ## access_log /var/log/nginx/artifactory-access.log timing;
+      ## error_log /var/log/nginx/artifactory-error.log;
+      rewrite ^/$ /artifactory/webapp/ redirect;
+      rewrite ^/artifactory/?(/webapp)?$ /artifactory/webapp/ redirect;
+      if ( $repo != "" ) {
+        rewrite ^/(v1|v2)/(.*) /artifactory/api/docker/$repo/$1/$2;
```

Comment:
Add the lines you added in `values.yaml`

Reply:
@eldada Done